src/test/java/SeleniumFramework.java

```diff
@@ -12,20 +12,25 @@
     public static void LoginSuccess() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
         driver.get("https://deens.com/?internaltraffic");
-        WebElement login_button = driver.findElement(By.cssSelector("a[href*='login']"));
-        login_button.click();
-        login_button.sendKeys("Mendygax@gmail.com");
+        driver.manage().window().maximize();
         Thread.sleep(1000);
 
-        WebElement password = driver.findElement(By.cssSelector("#password"));
-        password.click();
-        password.sendKeys("Positive89");
+        driver.findElement(By.cssSelector("a[href*='login']")).click();
+        Thread.sleep(1000);
+
+        driver.findElement(By.cssSelector("[id=\'email\']")).sendKeys("Mendygax@gmail.com");
+        driver.findElement(By.cssSelector("#password")).sendKeys("Positive89");
 
         Thread.sleep(1000);
 
         driver.findElement(By.cssSelector("[data-testid=\'loginSubmit\']")).click();
 
-        Thread.sleep(2000);
+        //Assert.assertTrue(driver.findElement(By.cssSelector("[class=\'ui error message\']")).isDisplayed());
+        Thread.sleep(3000);
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL, "https://deens.com/" );
```

Comment:
it's better to check some element after login(example profile image or user name if it displayed on the page)